intro-Linux/ch_navigating.tex

```diff
@@ -115,7 +116,7 @@ \section{Absolute vs relative file paths}
 with ``/''), which is also called the \emph{root} of the filesystem.
 
 Example: absolute path to your home directory:
-``%\homedir%''.
+``\homedir''.
```

Comment:
@redfast00 Ah, we should probably have used `\texttt{\homedir}` here?

Please fix that in some other PR where it makes sense (one that deal with typos/formatting in another chapter)